data/libs/autoload.lua

```diff
@@ -85,9 +85,15 @@ end
 ---@param predicate nil|fun(k: K, v: V): any, any
 ---@return table
 table.merge = function(a, b, predicate)
-	for k, v in pairs(b) do
-		if predicate then k, v = predicate(k, v) end
-		a[k] = v
+	if predicate then
```

Comment:
Sorry, change the name please. Predicate returns bool. Maybe "transform" will do?

Reply:
Done!